### DISCUSSION TITLE
8.2.5-Beta-3-Classic not Working Correctly

### DISCUSSION THREAD

**Discussion by qhilmv** [SFX-WoW/Masque#189] [Issues] [answered]
Hi there,

I just downloaded 8.2.5-Beta-3-Classic via Twitch App but it seems it's causing some issues. WeakAuras is not working correctly (no auras, no option menu, it hangs completely) and all my bars (Bartender4) are gone. No idea what happened but Beta-2 was working properly.

Even when I disable all add-ons but WeakAura (respectively Bartender) the new version of Masque doesnt' work.

Thanks,
qhilmv

**Comment by StormFX** [comment]
It's working properly on my end. Try reinstalling it.

**Comment by qhilmv** [comment, marked as answer]
I already did.
Okay, after some restarts of the game, everything works.

**Comment by StormFX** [comment]
Let me guess: You updated with the game running? :p

**Comment by qhilmv** [comment]
I'm on a server with a queue and I don't like it :)
Sorry for wasting your time. You're doing a great job!

**Comment by StormFX** [comment]
It's all good. I added some files, which wouldn't be loaded until restarting the game. Closing.
